apps/webservice/src/app/[workspaceSlug]/settings/(settings)/workspace/integrations/(integration)/github/GithubOrgConfig.tsx

```diff
@@ -58,6 +70,8 @@ export const GithubOrgConfig: React.FC<{
   const [value, setValue] = useState<string | null>(null);
   const [image, setImage] = useState<string | null>(null);
 
+  const baseUrl = api.runtime.baseUrl.useQuery();
```

Comment:
```suggestion
  const baseUrl = api.runtime.baseUrl.useQuery();
```

I though we would do something like runtime.useQuery() and it return an obj